src/Application.php

```diff
@@ -1538,6 +1556,32 @@ public function getRoutes()
     }
 
     /**
+     * Get the application namespace.
+     *
+     * @return string
+     *
+     * @throws \RuntimeException
+     */
+    public function getNamespace()
```

Comment:
As you can see in the issue some Illuminate components that Lumen uses require it .